khard/helpers.py

```diff
@@ -202,6 +202,10 @@ def get_new_contact_template(supported_private_objects=None):
 
     # create template
     return dedent("""
+        # Every contact must contain a formatted name, it will be autofilled
+        # from the full name below if not given.
+        Formatted name : 
```

Comment:
Isn't it rather `full name` when it's the input, and `formatted name` when it's generated by first name/last name templating?

Reply:
It represents the value in the `FN` field of the vcard.  And that is
> formatted text corresponding to the name

https://tools.ietf.org/html/rfc6350#section-6.2.1